Project.toml

```diff
@@ -27,6 +27,7 @@ GeoInterface = "cf35fbd7-0cd7-5166-be24-54bfbe79505f"
 Graphs = "86223c79-3864-5bf0-83f7-82e725a168b6"
 HypothesisTests = "09f84164-cd44-5f33-b23f-e6b0d136a0d5"
 ImageIO = "82e4d734-157c-48bb-816b-45c225c6df19"
+Infiltrator = "5903a43b-9cc3-4c30-8d17-598619ec4e9b"
```

Comment:
Debug/dev packages should only be in your sandbox environment and never added as dependencies to the main project spec.

I see the docs are light on detail on the proper use of these so that'll be addressed shortly.

For now, please remove.